src/containerapp/azext_containerapp/_clients.py

```diff
@@ -527,7 +527,7 @@ class GitHubActionClient():
     @classmethod
     def create_or_update(cls, cmd, resource_group_name, name, github_action_envelope, headers, no_wait=False):
         management_hostname = cmd.cli_ctx.cloud.endpoints.resource_manager
-        api_version = NEW_API_VERSION
+        api_version = "2022-03-01"
```

Comment:
Why hard code the version here instead of upgrading the version of `NEW_API_VERSION`?  We usually use one version according to the RP dimension, because it is easy to maintain

Reply:
@zhoxing-ms We will have 2 versions for a while until we test all commands with stable api-version '2022-03-01'
The NEW_API_VERSION points to the preview version & we will move out of this in time for GA. Once we have all tests covered we will make the change as a separate PR. A lot of the nee features we are adding support for will be with the non-preview version, like this updates to GitHub actions. Hope that explains.